docs/lib/getting-started/fragments/android/setup.md

```diff
@@ -1,78 +1,110 @@
-Build an Android app using the Amplify Framework. The Amplify Framework includes:
 
-- Amplify Tools - CLI toolchain for creating and managing your serverless backend.
-- Android, iOS, and JavaScript libraries to access your resources using a category based programming model.
-- Framework-specific UI component libraries for React, React Native, Angular, Ionic and Vue.  
+👋 Welcome! In this tutorial, you will:
 
-<br />
+- Create an Android application configured with Amplify
+- Use Amplify DataStore to model and persist data
+- Connect your local data to synchronize to a cloud backend
 
-This page guides you through setting up a backend and integrating Amplify into your Android app. You will create a "Note app" with a GraphQL API to store and retrieve items in a cloud database. The app will also receive updates over a realtime subscription.
+## Prerequisites
 
-[GraphQL](http://graphql.org) is a data language that was developed to enable apps to fetch data from APIs. It has a declarative, self-documenting style. In a GraphQL operation, the client specifies how to structure the data when it is returned by the server. This makes it possible for the client to query only for the data it needs, in the format that it needs it in.
+  <amplify-callout warning>
 
-## Prerequisites
+  **KNOWN LIMITATION** This tutorial currently does not work under Windows.
```

Comment:
I approve of the warning being this bold at the top - if there's no benefit for such a major group of users to continue on with the tutorial, we should make it clear sooner rather than later in the spirit of customer obsession. 

It's our bad that we don't support windows right now and in the spirit of Earns Trust let's not pretend our BO smells like perfume.

If we don't want to have this big warning, we should either not make Gradle-Tools the method used in the getting started doc (my preference) or get it fixed before GA release.

Reply:
The reaction you're having @wooj2 is the one I want us to have: we have a blight on our tutorial and need to fix it. The way to fix it isn't to be less direct in the language, but rather to fix it for Windows users. It does work on Linux.